scripts/pre-release.sh

```diff
@@ -79,8 +89,47 @@ function zipSamplesAddOn {
     delete_dir spartacussampledata
 }
 
+
+function zipEpdVisualizationSamplesAddOn {
+    echo "--> Generating EPD Visualization Spartacus sample data addon archives"
+    delete_dir epdvisualizationspartacussampledata
+    git clone https://github.tools.sap/cx-commerce/epdvisualizationspartacussampledata.git
```

Comment:
Please grant our [service user](https://github.tools.sap/spartacus-serviceuser) read access to your repo so that we can do this

Reply:
I have given the [https://github.tools.sap/spartacus-serviceuser](service user) read access and I have given @hackergil and @Platonn admin access.

Reply:
Update - I have given the service user write access which would make it possible to add tags in that script.